robot/rospackages/src/mcu_control/scripts/ArmNode.py

```diff
@@ -5,6 +5,7 @@
 import time
 import re
 
+from util import init_serial
```

Comment:
This should be `from robot.util....` and so on. Provide the full path from the root of the `robot` module.
Additionally lets improve the semantics of the rather ambigous `util` name to something like `serial_util` so that in the future all things related to serial functionality will be placed there.

Reply:
Actually since it's in mcu_control the proper import statement would have to be:
`robot.rospackages.src.mcu_control.scripts.serial_util`

Reply:
Fixed